File: 1450.py
# Author: Zoljargal Gantumur
# Runtime: 36 ms, faster than 82.16% of Python3 online submissions
# Memory Usage: 13.8 MB, less than 55.06% of Python3 online submissions

class Solution:
    def busyStudent(self, startTime: List[int], endTime: List[int], queryTime: int) -> int:
        count = 0
        for i in range(0, len(startTime)):
            if startTime[i] <= queryTime <= endTime[i]:
                count+=1
        return count

File: 1450.cpp
/*
	Author: Zoljargal Gantumur
	Runtime: 4 ms, faster than 90.10% of C++ online submissions
	Memory Usage: 11.1 MB, less than 42.70% of C++ online submissions
*/

class Solution {
public:
    int busyStudent(vector<int>& startTime, vector<int>& endTime, int queryTime) {
        int count = 0;
        for(int i=0; i<startTime.size(); i++){
            if(startTime[i]<=queryTime && queryTime<=endTime[i]){
                count++;
            }
        }
        return count;
    }
};
